.changeset/smooth-baboons-beg.md

```diff
@@ -0,0 +1,6 @@
+---
+'@commercetools-uikit/tag': minor
+---
+
+Add additional border-radius and color styling to Tag components.  
```

Comment:
In this case, we can be more generic so maybe something like this is good enough:
```
Component UI styling updates.
```